### ISSUE TITLE
Is there a "Dark OGC"? How do we share awareness and transparency between long range planning activities, rapid developments in the Innovation Program, and the detailed work in the Standards Program? Has OGC or a subset of members predetermined future paths that should be shared will all?

### ISSUE THREAD

**Issue by ogcscotts** [ogcscotts/TC-Meeting-topics#113]
- SMAC
- paraphrase as ”are things getting done outside the process until late and not reflecting consensus?”
- coined the phrase; work entering the Standards process too mature and with expectations of implementation.
- OGC was once accused of being too closed, we may have swung too far and let too much to happen outside our control. Impacts the value of membership.
- concern that sometimes there are not enough resources in Standards Program to capitalize on the work being done in Innovation Program activities
- we often struggle to get quorum on votes, is this a symptom
- voting has always been a problem, so not sure of the link
- surprises are inevitable with so many activities going on, probably largely a communication issue. There are initiatives run to try and get new capabilities faster. How do we increase engagement.
- bandwidth and visibility issue that needs to be solved. We do need to look at the value of voting membership
- lots of activities in IP, try to present what will happen. The information is available, so how do we make it more visible. Members do get to know about upcoming initiatives before non-members. Not all initiatives are open or visible to the public.
- as a SWG chair running into the problem with bandwidth – cannot work on standards unless have funding to execute the work. No problem in his perspective with IP – more people need to participate.
- worry about the “Alt OGC” – all the stuff that is out in the open that is not OGC. OGC is competing with other offerings (e.g., freely-available specs).
- I am Spartacus! OGC is trying to move to a more agile process with long term benefits, but short term pain at times. A value of membership is that someone is still needed to herd the cats.
- this is a common problem in the open source software community as well, if you want something in an OGC standard, join and participate
- not a bad thing that things are opened up. We can entice more people to participate.
not critiquing Innovation Program, bigger issue is keeping track of day-to-day activities. If an organization has interest in many things, impossible to keep track
- the value issue is interesting. OGC needs to express the right value to get the increase in bandwidth that we need
- not a unique problem to OGC!
- t used to be that we could come to OGC and get everything. World has changed and OGC needs to evolve. Know that the value proposition keeps changing.
- a value proposition for OGC could be to be the filter and aggregator of relevant expertise/tools for the community
- use STAC as an example. Not against the work, but part of the value of OGC is the competitive advantage of knowing what is coming? Those who are not part of the STAC work will not get that advantage if it comes suddenly into OGC
- found ourselves in circumstances that happened for all of the right reasons, but the processes in OGC have not evolved with these changes. We should step back and reassess the value of OGC membership in this more open context.
- is there a disconnect between organizations more heavily involved in SP vs. IP and some involved in both?
- probably goes back to the need to modernize process

**Comment by jyutzler** [comment]
This is a complicated topic that, as was noted, transcends OGC. 

Something that I didn't think of last week that could be beneficial is better communication of the point of the engineering reports and discussion papers that come out of the Innovation Program and other initiatives. While I believe the quality of these work products has improved over the last couple of years, they often take a big commitment to grok. I'm probably not going to read your 50-page paper and I wouldn't be surprised if you aren't going to read mine either.

Maybe it is just me, but I find most formal papers impenetrable. I shed my ACM and similar memberships years ago even though research is a major component of what I do. Perhaps making formal executive summaries as part of the process would help. The abstracts are usually too thin so I typically have to read the bulk of the paper to figure out what the point is. Something on the order to 1/2 to 1 page per paper might go along way towards increasing visibility of these activities. You can get me to read that and I might be able to get you to read that as well.

**Comment by RyanAhola** [comment]
> This is a complicated topic that, as was noted, transcends OGC.
> 
> Something that I didn't think of last week that could be beneficial is better communication of the point of the engineering reports and discussion papers that come out of the Innovation Program and other initiatives. While I believe the quality of these work products has improved over the last couple of years, they often take a big commitment to grok. I'm probably not going to read your 50-page paper and I wouldn't be surprised if you aren't going to read mine either.
> 
> Maybe it is just me, but I find most formal papers impenetrable. I shed my ACM and similar memberships years ago even though research is a major component of what I do. Perhaps making formal executive summaries as part of the process would help. The abstracts are usually too thin so I typically have to read the bulk of the paper to figure out what the point is. Something on the order to 1/2 to 1 page per paper might go along way towards increasing visibility of these activities. You can get me to read that and I might be able to get you to read that as well.

I completely agree with jyutzler's suggestion of developing executive summaries for OGC publications. As sponsors of some OGC initiatives, Natural Resources Canada staff constantly need to articulate the value of OGC work to their executives. If our executives do not clearly understand the benefit of OGC activities to the department's priorities, they are less likely to support sponsorship of future initiatives. Summarizing the main findings of OGC work and how these can be practically applied in the real world would greatly help us with this type of communication. Use of plain language will be key.
